matrix_client/api.py

```diff
@@ -238,6 +238,16 @@ def send_message(self, room_id, text_content, msgtype="m.text"):
             self.get_text_body(text_content, msgtype)
         )
 
+    def send_html(self, room_id, html_content, msgtype="m.text"):
+        """Send an html formatted message.
+
+        Args:
+            room_id (str): The room ID to send the event in.
+            html_content (str): The html formatted message to be sent.
+        """
+        return self.send_message_event(room_id, "m.room.message",
+                                       self.get_html_body(html_content, msgtype))
```

Comment:
Oh, I think I mistake `body` for `content` 😕, don't mind me.
Now it all makes sense 😆.